stack/testParameters/PublicIPsWithSyncGateway.json

```diff
@@ -21,7 +21,7 @@
       "value": true
     },
     "scriptsUri": {
-        "value": "https://raw.githubusercontent.com/couchbase-partners/azure-resource-manager-couchbase/master/stack/"
+        "value": "https://raw.githubusercontent.com/malscent/azure-resource-manager-couchbase/master/stack/"
```

Comment:
Is this back to 'malscent' for testing purposes?

Reply:
Yeah.. i try to put those back before I pr.. but i missed that one